.env

```diff
@@ -0,0 +1,2 @@
+COMPOSE_PROJECT_NAME=data_pipeline
```

Comment:
I think `-` are also allowed now (we usually stick to `_` to separate various image of the same project e.g. `pattern-library_assets`

Reply:
Okay, didn't know that. I will change it to a hyphen